### PR TITLE
fix cgroup.devices_access

### PR DESCRIPTION
**Pull request by hjydxy**
In qemu1.5,we get the information like "idTestR2: removable=1 locked=0..."
In new version,we get the information like "idTestR2 (#block303): /dev/sda"
The most important thing is weather "idTestR2" in the information.
The expression ("%s: " % name) won't work for the new version.
This modify is compitable for old version.

### CODE REVIEW

**Comment by xutian** [comment]
git commit -s --amend to add sign info please .

**Comment by hjydxy** [comment]
@xutian Is that so？

File: qemu/tests/cgroup.py [hunk 1]
```diff
@@ -169,27 +170,27 @@ def rm_scsi_disks(no_disks):
...
-        if utils.system("lsmod | grep scsi_debug", ignore_status=True):
-            utils.system("modprobe scsi_debug dev_size_mb=8 add_host=0")
+        if process.system("lsmod | grep scsi_debug", ignore_status=True):
+            process.system("modprobe scsi_debug dev_size_mb=8 add_host=0")
         for name in params['vms'].split(' '):
             disk_name = "scsi-debug-" + name
             utils.system("echo 1 >/sys/bus/pseudo/drivers/scsi_debug/add_host")
             time.sleep(1)   # Wait for device init
-            dev = utils.system_output("ls /dev/sd* | tail -n 1")
+            dev = process.system_output("ls /dev/sd* | tail -n 1")
```
Comment (balamuruhans) [inline qemu/tests/cgroup.py]: `dev = process.system_output("ls /dev/sd* | tail -n 1", shell=True)` else you might hit,
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/avocado_framework-46.0-py2.7.egg/avocado/utils/process.py", line 1211, in system_output
    sudo=sudo, ignore_bg_processes=ignore_bg_processes)
  File "/usr/lib/python2.7/site-packages/avocado_framework-46.0-py2.7.egg/avocado/utils/process.py", line 1111, in run
    raise CmdError(cmd, sp.result)
```

File: qemu/tests/cgroup.py [hunk 2]
```diff
@@ -169,27 +170,27 @@ def rm_scsi_disks(no_disks):
...
         Adds scsi_debug disk to every VM in params['vms']
         :param prefix: adds prefix to drive name
         """
-        if utils.system("lsmod | grep scsi_debug", ignore_status=True):
-            utils.system("modprobe scsi_debug dev_size_mb=8 add_host=0")
+        if process.system("lsmod | grep scsi_debug", ignore_status=True):
+            process.system("modprobe scsi_debug dev_size_mb=8 add_host=0")
         for name in params['vms'].split(' '):
             disk_name = "scsi-debug-" + name
             utils.system("echo 1 >/sys/bus/pseudo/drivers/scsi_debug/add_host")
```
Comment (balamuruhans) [inline qemu/tests/cgroup.py]: please use `process` instead of `utils` here too

File: qemu/tests/cgroup.py [hunk 3]
```diff
@@ -1148,17 +1149,17 @@ def _generate_verification(cpusets, cpus):
...
-            error.context("Generating cpusets expected results")
+            logging.info("Generating cpusets expected results")
             try:
                 verify = _generate_verification(cpusets, cpus)
             except IndexError:
-                raise error.TestError("IndexError occurred while generatin "
-                                      "verification data. Probably missmatched"
-                                      " no_host_cpus and cgroup_cpuset cpus")
+                raise exceptions.TestError("IndexError occurred while generatin "
+                                           "verification data. Probably missmatched"
```
Comment (balamuruhans) [inline qemu/tests/cgroup.py]: `%s/generatin/generating` and `%s/missmatched/mismatched`

File: qemu/tests/cgroup.py (hunk 2 above)
Comment (balamuruhans) [inline qemu/tests/cgroup.py]: `L310 and L514- utils.matrix_to_string()`

**Comment by hjydxy** [comment]
@xutian  @nasastry @balamuruhans    All fixed, thanks for your patient guidence.

**Comment by hjydxy** [comment]
@xutian @balamuruhans @nasastry All fixed, thanks for your guidance.

File: qemu/tests/cgroup.py [hunk 4]
```diff
@@ -306,8 +307,8 @@ def _test(direction):
                     err += "%d, " % i
 
             logging.info("blkio_bandwidth_%s: dd statistics\n%s", direction,
-                         utils.matrix_to_string(out, ['status', 'norm_weights',
-                                                      'norm_out', 'actual']))
+                         process.matrix_to_string(out, ['status', 'norm_weights',
```
Comment (balamuruhans) [inline qemu/tests/cgroup.py]: just noticed that API matrix_to_string() is not available in avocado, possible solution is
1. Earlier, you have used utils.matrix_to_string() but utils was not imported led to error - so you can use utils.matrix_to_string() by importing utils.
2. If the API can be used by many test in future, can be added to avocado-vt and then used in your test.

**Comment by hjydxy** [comment]
@balamuruhans   Use utils for now.When matrix_to_string() is available in avocado, i'll change it to process. matrix_to_string().Is that ok?

**Comment by balamuruhans** [comment]
@hjydxy okay, Looks good.

**Comment by hjydxy** [comment]
@balamuruhans Thank you! Please merge this PR if it's ok.

**Comment by hjydxy** [comment]
Ping....

**Comment by balamuruhans** [comment]
@hjydxy Thanks for the patch

**Pull request merged**